packages/grafana-ui/src/components/Table/TableNG/Cells/MarkdownCell.tsx

```diff
@@ -6,12 +6,19 @@ import { MaybeWrapWithLink } from '../MaybeWrapWithLink';
 import { MarkdownCellProps, TableCellStyles } from '../types';
 
 export function MarkdownCell({ field, rowIdx, disableSanitizeHtml }: MarkdownCellProps) {
+  const rawValue = field.values[rowIdx];
+  if (rawValue === null || rawValue === undefined) {
```

Comment:
`== null` is faster and covers both cases in a single comparison, also shorter to read. the perf in this case will be negligible, but it the single comparison does move the needle when large arrats of data in a loop. i usually just use `== null` in all cases for uniformity.